capsule.bxb

```diff
@@ -4,13 +4,17 @@ capsule {
   version (0.1.1)
   targets {
     target (bixby-mobile-en-US)
-    target (bixby-mobile-ko-KR){enabled (false)}
+    target (bixby-mobile-ko-KR) {
+      enabled (false)
+    }
+  }
+  runtime-version (2) {
```

Comment:
Is there a reason we're using `runtime-version(2)` instead of a more recent one?

Reply:
probably not. @fdsgusmao can we update to the latest runtime version and run the tests again to see if there are any issues?